types/capabilities.ts

```diff
@@ -0,0 +1,6 @@
+export enum Capabilities {
```

Comment:
shouldnt this be in some format that works cross language? like a JSON file or something?

Reply:
yeah youre right lol.